src/platform/site-wide/header/components/MenuItemLevel1/index.js

```diff
@@ -0,0 +1,133 @@
+// Node modules.
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+// Relative imports.
+import MenuItemLevel2 from '../MenuItemLevel2';
+import { deriveMenuItemID, formatMenuItems } from '../../helpers';
+import { updateExpandedMenuIDAction } from '../../containers/Menu/actions';
+
+export const MenuItemLevel1 = ({
+  expandedMenuID,
+  item,
+  updateExpandedMenuID,
+}) => {
+  // Do not render if we are missing necessary menu item data.
+  if (!item?.menuSections && !item?.href && !item?.title) {
+    return null;
+  }
+
+  // Format menuSections to be an array if needed.
+  if (item?.menuSections) {
+    item.menuSections = formatMenuItems(item.menuSections); // eslint-disable-line no-param-reassign
+  }
+
+  // Derive the menu item's ID.
+  const menuItemID = deriveMenuItemID(item, '1');
+
+  // Derive if we the menu item is expanded.
+  const isExpanded = menuItemID === expandedMenuID;
+
+  const toggleShowItems = () => {
+    // Update the expanded menu ID.
+    updateExpandedMenuID(isExpanded ? undefined : menuItemID);
+
+    // Derive the header element.
+    const header = document.querySelector('header');
+
+    // Scroll to the top of the header when toggling a menu item.
+    if (header) {
+      header.scrollIntoView();
+    }
+  };
+
+  return (
+    <li
+      className="vads-u-background-color--primary-darker vads-u-margin--0 vads-u-margin-bottom--0p5 vads-u-width--full vads-u-font-weight--bold"
+      role="menuitem"
```

Comment:
Can remove menuitem as is doesn't apply to navigation

Reply:
Totally! This was brought over from the current megamenu's implementation 🙂

Reply:
All role="menu" role="menuitem"s removed 🙂